application/lib/test/phantom_subnets.toml

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
How do we keep this file synced with clientConfs? Does it make sense to make a script that takes a (set of or new) clientConf and produces this .toml (or a delta to an existing one)? Happy to make such a script if we have clientConfs ready

Reply:
This will happen in the decoy-list system for managing the decoy & phantom sets. The first step is getting these set up as static files (and a static clientconf) to get clients and stations working together. Next is to update the decoy-lists system with scripts to sync / update the tracked clientconfs and phantoms list based new sets / clientconfs and produces a new .toml file.  That way we can push to the repo in order to update clientconfs and station generation tracking together.